test/integration/als.test.ts

```diff
@@ -51,10 +51,10 @@ describe('AsyncLocalStorage Transactions (ALS)', () => {
       let t2id;
       await Promise.all([
         vars.alsSequelize.transaction(async () => {
-          t1id = vars.alsSequelize.getCurrentAlsTransaction()!.id;
+          t1id = vars.alsSequelize.getCurrentClsTransaction()!.id;
```

Comment:
als is still used a few times in this file

Reply:
Done